### PR TITLE
#6 Maya v1.0.0 UI overview

### PR DESCRIPTION
**Pull request by luisalbaceteinbibo**
### Description of this issue
In this issue we document the Maya overview page with the information about the shelf and menu.

### Pages to review:
- `ui_overview.md`

Issue of this PR: #6

### CODE REVIEW

File: docs/documentation/maya/ui_overview.md [hunk 1]
```diff
@@ -99,20 +99,20 @@ Option boxes might be used for deformer creation. Openning an auxiliar window th
 - **Remove Deform Mesh**. Removes a [Deform Mesh](simshape.md#requirements) from the deformer selected. The selection must consist of: first the Deform Mesh that we want to remove and lastly the mesh with the deformer applied.
```
Comment (lucasferryinbibo) [inline docs/documentation/maya/ui_overview.md]: I know this information might be redundant as it is exposed in the table but we might want to also add the information related to adding the collider and removing the collider

File: docs/documentation/maya/ui_overview.md [hunk 2]
```diff
@@ -99,20 +99,20 @@ Option boxes might be used for deformer creation. Openning an auxiliar window th
...
 <figure markdown>
-  ![AdonisFX Menu Edit Simshape](images/adonis_menu_simshape.png){width=30%}
-  <figcaption><b>Figure 8:</b> AdonisFX Menu Edit Simshape section.</figcaption>
+  ![AdonisFX Menu Edit AdnSimshape](images/adonis_menu_simshape.png){width=30%}
+  <figcaption><b>Figure 8:</b> AdonisFX Menu Edit AdnSimshape section.</figcaption>
 </figure>
 
 #### Muscle
 
 - **Add [Slide On Segment](muscle.md#slide-on-segment-constraint) Constraint**. Adds segments based on the two (or more) transform objects and the mesh with the deformer node applied selected.
```
Comment (lucasferryinbibo) [inline docs/documentation/maya/ui_overview.md:112]: The same applies here, do we want to explain all entries in the menu?

**Review by lucasferryinbibo — APPROVED** [review]
Just two comments to check

File: docs/documentation/maya/ui_overview.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # UI Overview
 
-The Adonis UI for maya can be separated into two main elements: the **AdonisFX Shelf** and the **AdonisFX Menu**. Many of the uses here we will go into detail in both of them to explain what every element of the UI does.
+The Adonis UI for maya can be separated into two main elements: the **AdonisFX Shelf** and the **AdonisFX Menu**. The many uses the UI elements have will be explained in this page.
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/ui_overview.md:3]:
```suggestion
The Adonis UI for Maya can be separated into two main elements: the **AdonisFX Shelf** and the **AdonisFX Menu**. The many uses the UI elements have will be explained in this page.
```

File: docs/documentation/maya/ui_overview.md [hunk 4]
```diff
@@ -91,30 +91,30 @@ Option boxes might be used for deformer creation. Openning an auxiliar window th
...
 </figure>
 
 #### Muscle
 
 - **Add [Slide On Segment](muscle.md#slide-on-segment-constraint) Constraint**. Adds segments based on the two (or more) transform objects and the mesh with the deformer node applied selected.
 - **Remove [Slide On Segment](muscle.md#slide-on-segment-constraint) Constraint**. Removes anchors from segments based on the transform objects and the mesh with the deformer node applied selected.
-- **Draw Fibers**. Enables "*Muscle Fibers*" Debugging from all the [Ribbon Muscles](ribbon.md#debug-features) an [Muscles](muscle.md#debug-features) found in the scene.
-- **Hide Fibers**. Disables "*Muscle Fibers*" Debugging from all the [Ribbon Muscles](ribbon.md#debug-features) an [Muscles](muscle.md#debug-features) found in the scene that were debugging "*Muscle Fibers*".
+- **Draw Fibers**. Enables "*Muscle Fibers*" Debugging from all the [AdnRibbonMuscle](ribbon.md#debug-features) an [AdnMuscle](muscle.md#debug-features) found in the scene.
+- **Hide Fibers**. Disables "*Muscle Fibers*" Debugging from all the [AdnRibbonMuscle](ribbon.md#debug-features) an [AdnMuscle](muscle.md#debug-features) found in the scene that were debugging "*Muscle Fibers*".
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/ui_overview.md:115]:
```suggestion
- **Draw Fibers**. Enables *Muscle Fibers* Debugging from all the [AdnRibbonMuscle](ribbon.md#debug-features) an [AdnMuscle](muscle.md#debug-features) found in the scene.
- **Hide Fibers**. Disables *Muscle Fibers* Debugging from all the [AdnRibbonMuscle](ribbon.md#debug-features) an [AdnMuscle](muscle.md#debug-features) found in the scene that were debugging *Muscle Fibers*.
```

File: docs/documentation/maya/ui_overview.md [hunk 5]
```diff
@@ -13,46 +13,46 @@ The AdonisFX shelf can be found in the Maya shelf tab under the label "*AdonisFX
...
+| ![Add AdnSimshape Collider](images/adn_add_collider.png) | [Add Collider to AdnSimshape](simshape.md#add-collider), Assigns a collider to the AdnSimshape deformer. Selecting first, in order, collider and mesh with AdnSimshape applied. | AdonisFX Menu > Edit Simshape > *Add Collider* |
+| ![Remove AdnSimshape Collider](images/adn_remove_collider.png) | [Remove Collider from AdnSimshape](simshape.md#remove-collider), removes the collider from the AdnSimshape deformer. Selecting first, in order, collider assigned and mesh with AdnSimshape applied. | AdonisFX Menu > Edit Simshape > *Remove Collider* |
+| ![AdnSimshape Learn Muscle Patches Tool](images/adn_learn_muscle_patches.png) | Launches the [AdnSimshape Muscle Patches Learning](simshape.md#generate-muscle-patches) tool used to generate the "*Adonis Muscle Patches*" file (.amp) file. | AdonisFX Menu > Edit Simshape > *Learn Muscle Patches UI* |
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/ui_overview.md:28]:
```suggestion
| ![AdnSimshape Learn Muscle Patches Tool](images/adn_learn_muscle_patches.png) | Launches the [AdnSimshape Muscle Patches Learning](simshape.md#generate-muscle-patches) tool used to generate the *Adonis Muscle Patches* file (.amp) file. | AdonisFX Menu > Edit Simshape > *Learn Muscle Patches UI* |
```

**Review by josevalverdeinbibo — APPROVED** [review]
Great job! Small comments to check.

File: docs/documentation/maya/ui_overview.md (hunk 3 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/ui_overview.md:3]: Done.

File: docs/documentation/maya/ui_overview.md (hunk 4 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/ui_overview.md:115]: Done.

File: docs/documentation/maya/ui_overview.md (hunk 5 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/ui_overview.md:28]: Done.

**Pull request merged**